topics/unreal-engine/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: ue4, unrealengine, unrealengine4, unreal-engine-4
+created_by: Epic Games
+display_name: Unreal Engine
+github_url: https://github.com/EpicGames/
+logo: unreal-engine.png
+short_description: Developers choose Unreal Engine to create awesome games, and experiences for PC, mobile, console, VR, and AR.
```

Comment:
This might sound a bit accidentally sales-y 😄 How about rephrasing to _Unreal Engine is..._, similar to the longer description?

Reply:
@nayafia let it sell :wink: